### PR TITLE
Updated de (German) translation

### PR DESCRIPTION
**Pull request by squeezer11**

### CODE REVIEW

**Comment by Vankog** [comment]
Hey @squeezer11, thx for the update.
However, I have several requests for you.
1. Could you please match your contribution with this one: #965 
1. Please delete the string_preferences and string_static files. These are not intended to be translated. (see: #969)

**Comment by squeezer11** [comment]
my translations are making now a lot of more sense than before and a much better understanding. hope i have reached your goal by finishing the german translation.

File: AdAway/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -68,99 +68,97 @@
...
+    \n
+  Eine Lösung besteht darin, das Telefon in den Wiederherstellungsmodus zu starten und den Symlink dort mit \'ln -s / data/ data/hosts/system/etc/hosts\' zu erstellen. \n
+    \n
+  Wenn dies nicht funktioniert, suchen Sie im Internet nach S-OFF und Ihrem Telefon.</string>
+    <string name="apply_symlink_fail_title">Symlink-Erstellung fehlgeschlagen</string>
+    <string name="apply_help">Bitte lesen Sie die Hilfe für weitere Informationen.</string>
+    <string name="apply_apn_proxy">Ein APN-Proxy ist auf Ihrem Android eingestellt! AdAway funktioniert nicht zuverlässig in mobilen Netzwerken wie 3G. Sie können diesen Proxy deaktivieren, indem Sie zum ausgewählten APN wechseln (unter Android 4: Drahtlos und Netzwerke -> Mehr ... -> Mobile Netzwerke -> Zugangspunktnamen) und den Wert im Proxy-Feld entfernen.</string>
     <string name="apply_apn_proxy_title">APN-Proxy eingestellt!</string>
-    <string name="no_connection">Die Internetverbindung funktioniert nicht.</string>
+    <string name="no_connection">Die Verbindung zum Internet funktioniert nicht.</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Why is this better? It is basically the same thing but longer. Also, why not go with `Die Verbindung zum Internet ist unterbrochen.` or something similar?

File: AdAway/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -68,99 +68,97 @@
...
     <string name="no_connection_title">Keine Verbindung</string>
     <string name="download_dialog">Wird heruntergeladen …</string>
     <string name="apply_dialog">Wird angewendet …</string>
     <string name="apply_dialog_lists">Negativ- und Positivliste werden angewendet</string>
-    <string name="apply_dialog_hostnames">Hosts-Dateien werden verarbeitet und zusammengefügt</string>
-    <string name="apply_dialog_hosts">Hosts-Datei wird erstellt</string>
+    <string name="apply_dialog_hostnames">Analysiere und führe Hostdateien zusammen</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again, revert or use `…`.

File: AdAway/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -68,99 +68,97 @@
...
+    <string name="apply_dialog_hosts">Erstelle Host-Datei</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Aaand again: `Hosts-Datei` and signaling of ongoing process by `…` or the use of passive.

File: AdAway/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -68,99 +68,97 @@
...
     <string name="apply_dialog_apply">Neue Hosts-Datei im System aktivieren</string>
-    <string name="apply_fail">Das Ändern der Hosts-Datei auf Ihrem System ist fehlgeschlagen!\n
-  Versuchen Sie bitte die Hosts-Datei, in den Einstellungen, auf /data/data/hosts zu ändern.</string>
+    <string name="apply_fail">Anwenden der Host-Datei auf Ihr System fehlgeschlagen! Versuchen Sie, die Zielhostdatei in den Einstellungen in /data/data/hosts zu ändern.</string>
     <string name="apply_fail_title">Anwendung fehlgeschlagen</string>
-    <string name="revert_successful">Die Änderungen wurden rückgängig gemacht.\n
-  Damit alle Änderungen wirksam werden, muss das System eventuell neu gestartet werden.\n
-  Möchten Sie neu starten?\n
-  (Um Datenverlust zu vermeiden, vergewissern Sie sich, dass keine Anwendung gerade die SD-Karte verwendet!)</string>
+    <string name="revert_successful">Sie haben die Standard-Hosts-Datei wiederhergestellt. Es wird empfohlen, Android Neu zu Starten, damit die Änderungen wirksam werden. \n
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again that `neu starten`. I hope I am not wrong on this one, but it just seems wrong.

File: AdAway/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -68,99 +68,97 @@
...
+    \n
+  Möchten Sie Neu Starten? (Um Datenverlust zu vermeiden, vergewissern Sie sich, dass keine App die Speicherkarte nutzt!)</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again `neu starten`.

File: AdAway/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -68,99 +68,97 @@
...
     <string name="revert_successful_title">Erfolgreich zurückgesetzt</string>
-    <string name="revert_problem">Wiederherstellen fehlgeschlagen.</string>
-    <string name="revert_problem_title">Die Wiederherstellung ist fehlgeschlagen!</string>
-    <string name="download_fail_dialog">Keine der aktivierten Hosts-Quellen ist erreichbar! Funktioniert Ihre Internetverbindung richtig?</string>
+    <string name="revert_problem">Das Zurücksetzen ist aus unbekannten Gründen fehlgeschlagen.</string>
+    <string name="revert_problem_title">Zurücksetzen fehlgeschlagen!</string>
+    <string name="download_fail_dialog">Keine der aktivierten Hostquellen ist erreichbar! Sind Sie richtig mit dem Internet verbunden?</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Would delete `richtig`. The relevant information for the user is that there might be a problem with his internet-connection.

File: AdAway/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -68,99 +68,97 @@
...
     <string name="download_fail_title">Herunterladen fehlgeschlagen</string>
-    <string name="no_private_file">Private Datei konnte nicht erstellt werden.</string>
+    <string name="no_private_file">Privat-Datei konnte nicht erstellt werden.</string>
     <string name="no_private_file_title">Erstellen der privaten Datei fehlgeschlagen</string>
-    <string name="enable_systemless_successful">Du hast den Systemlosen Modus aktiviert. Möglicherweise musst Du Android neu starten, damit die Änderungen wirksam werden.\n
- \n
- Möchtest Du neu starten? (Um Datenverlust zu vermeiden, vergewissere Dich, dass keine App die SD Karte im Moment benutzt!)</string>
+    <string name="enable_systemless_successful">Sie haben den Systemlosen Modus aktiviert.\n
+  Es wird Empfohlen, Android Neu zu Starten, damit die Änderungen wirksam werden. \n
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again, `empfohlen` and `neu starten`

File: AdAway/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -68,99 +68,97 @@
...
+    \n
+  Möchten Sie Neu Starten? \n
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: `Möchten Sie jetzt neu starten?`

File: AdAway/src/main/res/values-de/strings.xml [hunk 9]
```diff
@@ -68,99 +68,97 @@
...
+  (Um Datenverlust zu vermeiden, stellen Sie sicher, dass keine App die Speicherkarte nutzt!)</string>
     <string name="enable_systemless_successful_title">Aktivierung erfolgreich</string>
-    <string name="disable_systemless_successful">Sie haben den Systemlosen Modus deaktiviert.\n
-  Damit alle Änderungen wirksam werden, muss das System eventuell neu gestartet werden.\n
-  \n
-  Möchten Sie neu starten?\n
-  (Um Datenverlust zu vermeiden, vergewissern Sie sich, dass keine Anwendung gerade die SD-Karte verwendet!)</string>
+    <string name="disable_systemless_successful">Sie haben den systemlosen Modus deaktiviert.\n
+  Es wird Empfohlen, Android Neu zu Starten, damit die Änderungen wirksam werden. \n
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Same here.

File: AdAway/src/main/res/values-de/strings.xml [hunk 10]
```diff
@@ -68,99 +68,97 @@
...
+    \n
+    Möchten Sie Neu Starten? (Um Datenverlust zu vermeiden, stellen Sie sicher, dass keine App die Speicherkarte nutzt!)</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again.

File: AdAway/src/main/res/values-de/strings.xml [hunk 11]
```diff
@@ -68,99 +68,97 @@
...
     <string name="no_ip_title">Falsch formatierte IP</string>
     <string name="reboot_never_reboot">Niemals neu starten und diese Frage nicht wieder stellen!</string>
     <string name="loading">Wird geladen …</string>
 
     <!-- Menu entries -->
     <string name="menu_refresh">Neu laden</string>
     <string name="menu_add">Hinzufügen</string>
-    <string name="menu_scan_adware">Nach Adware suchen</string>
-    <string name="menu_hosts_sources">Hosts-Quellen</string>
+    <string name="menu_scan_adware">Suche nach Adware</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again, the infinitive is standard, why deviate.

File: AdAway/src/main/res/values-de/strings.xml [hunk 12]
```diff
@@ -68,99 +68,97 @@
...
+    <string name="menu_hosts_sources">Host-Quellen</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: File is called `Hosts`

File: AdAway/src/main/res/values-de/strings.xml [hunk 13]
```diff
@@ -68,99 +68,97 @@
...
     <string name="menu_tcpdump">DNS-Anfragen protokollieren</string>
     <string name="menu_help">Hilfe</string>
-    <string name="menu_lists">Meine Listen</string>
-    <string name="menu_show_hosts_file">Hosts-Datei öffnen</string>
+    <string name="menu_lists">Deine Listen</string>
+    <string name="menu_show_hosts_file">Öffne Host-Datei</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again, infinitive is the standard.

File: AdAway/src/main/res/values-de/strings.xml [hunk 14]
```diff
@@ -184,9 +182,9 @@
     </string-array>
 
     <!-- Scan Adware -->
-    <string name="scan_adware_description">Hier können Sie nach Adware suchen, die nicht von AdAway blockiert werden kann. Diese Anwendungen belästigen den Nutzer beispielsweise durch Airpush-Benachrichtigungen, die auch, wenn die Anwendung gerade nicht genutzt wird, in der Benachrichtungsleiste angezeigt werden. Gefundene Anwendungen können mit einem Klick auf den Eintrag deinstalliert werden!</string>
+    <string name="scan_adware_description">Hier können Sie nach bösen Adware-Anwendungen suchen, die nicht von AdAway geblockt werden können. Diese Apps verwenden beispielsweise Airpush-Benachrichtigungen, die selbst dann angezeigt werden, wenn die App nicht ausgeführt wird, oder sogar den Klingelton ändern. Die einzige verfügbare Gegenmaßnahme besteht darin, diese Apps zu deinstallieren, indem Sie auf die Listenelemente klicken!</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: `bösartiger Adware` is the phrase to use here. The word adware already implies that it is an application.

File: AdAway/src/main/res/values-de/strings.xml [hunk 15]
```diff
@@ -184,9 +182,9 @@
...
     <string name="scan_adware_start">Durchsuchen starten</string>
-    <string name="scan_adware_empty">Es wurde keine Adware gefunden!</string>
+    <string name="scan_adware_empty">Es wurden Keine Bösen Adware Apps gefunden!</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: `keine bösartige Adware`

Groß- und Kleinschreibung

File: AdAway/src/main/res/values-de/strings.xml [hunk 16]
```diff
@@ -195,44 +193,44 @@
 
     <!-- Other -->
     <string name="app_subtitle">Werbeblocker</string>
-    <string name="no_text_editor_title">Keine Textbearbeitung installiert</string>
-    <string name="no_text_editor">Es konnte keine Textbearbeitung gefunden werden, um die Hosts-Datei zu öffnen. Sie können Jota oder eine andere Textbearbeitung installieren, um Textdateien zu öffnen.\n
+    <string name="no_text_editor_title">Kein Texteditor installiert</string>
+    <string name="no_text_editor">Es konnte kein Texteditor gefunden werden, um die Hostdatei zu öffnen. Sie können Jota oder andere Texteditoren installieren, um damit zu interagieren. \n
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Should be reverted. Replacing `Textbearbeitung` with `Texteditor` is good, but everything else is worse imho.

File: AdAway/src/main/res/values-de/strings.xml [hunk 17]
```diff
@@ -195,44 +193,44 @@
...
   \n
   Möchten Sie Jota installieren?</string>
-    <string name="no_file_manager_title">Keine Dateiverwaltung installiert</string>
-    <string name="no_file_manager">Es konnte keine Dateiverwaltung gefunden werden, um Dateien zu öffnen. Sie können »OI File Manager« oder  andere Dateimanager installieren, um das zu beheben.\n
-  \n
-   Möchten Sie »OI File Manager« installieren?</string>
+    <string name="no_file_manager_title">Kein Datei-Manager installiert</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Should be unified to `Dateimanager`.

File: AdAway/src/main/res/values-de/strings.xml [hunk 18]
```diff
@@ -195,44 +193,44 @@
...
+    <string name="no_file_manager">Es konnte kein Dateimanager zum Öffnen von Dateien gefunden werden. Sie können den OI-Dateimanager oder andere Dateimanager installieren, um damit zu agieren.\n
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Same as above. The word `Dateimanager` is better than `Dateiverwaltung` but the other improvements are making things more complicated to understand.

File: AdAway/src/main/res/values-de/strings.xml [hunk 19]
```diff
@@ -195,44 +193,44 @@
...
 
     <!-- Tcpdump -->
     <string name="toast_tcpdump_log_deleted">Protokolldatei wurde gelöscht!</string>
-    <string name="toast_tcpdump_added_to_blacklist">Der Hostname wurde zu Ihrer Negativliste hinzugefügt!</string>
-    <string name="toast_tcpdump_added_to_whitelist">Der Hostname wurde zu Ihrer Positivliste hinzugefügt!</string>
+    <string name="toast_tcpdump_added_to_blacklist">Hostname wurde zu Ihrer Negativliste hinzugefügt!</string>
+    <string name="toast_tcpdump_added_to_whitelist">Hostname wurde zu Ihrer Positivliste hinzugefügt!</string>
     <string name="tcpdump_log_context_blacklist">Zur Negativliste hinzufügen</string>
-    <string name="tcpdump_log_context_whitelist">Zur Positivliste hinzufügen</string>
+    <string name="tcpdump_log_context_whitelist">Zu Deiner Positivliste hinzufügen</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Either revert or also change the string about `Negativliste` respectively. Should be unified.

File: AdAway/src/main/res/values-de/strings.xml [hunk 20]
```diff
@@ -195,44 +193,44 @@
...
     <string name="tcpdump_log_context_browser">Im Browser öffnen</string>
     <string name="tcpdump_log_empty">Protokolldatei ist leer!</string>
-    <string name="tcpdump_log_footer">Einen Eintrag lange gedrückt halten, um ihn zur Negativ- oder Positivliste hinzuzufügen!</string>
+    <string name="tcpdump_log_footer">Drücken Sie lange auf einen Hostnamen, um ihn zu Ihrer Negativ- oder Positivliste hinzuzufügen!</string>
 
     <!-- Preferences -->
-    <string name="pref_download">Grundeinstellungen</string>
-    <string name="pref_hosts">Hosts-Datei</string>
+    <string name="pref_download">Allgemeine Einstellungen</string>
+    <string name="pref_hosts">Host-Datei</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again, its `Hosts-Datei`

File: AdAway/src/main/res/values-de/strings.xml [hunk 21]
```diff
@@ -195,44 +193,44 @@
...
     <string name="pref_redirection_ip">Umleitungs-IP</string>
-    <string name="pref_update_check">Nach Aktualisierungen suchen</string>
+    <string name="pref_update_check">Suche nach Aktualisierungen</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again infinitive is standard. Should revert.

File: AdAway/src/main/res/values-de/strings.xml [hunk 22]
```diff
@@ -195,44 +193,44 @@
...
     <string name="pref_apply_method_summary">Bevor Sie diese Einstellung ändern, bitte die Hilfe zu dem Thema lesen.</string>
-    <string name="pref_redirection_ip_summary">Alle geblockten Hostnamen werden auf diese IP-Adresse umgeleitet.</string>
+    <string name="pref_redirection_ip_summary">Alle blockierten Hostnamen werden auf diese IP-Adresse umgeleitet.</string>
     <string name="pref_header_preferences">Einstellungen</string>
-    <string name="pref_header_preferences_summary">Generelle AdAway-Einstellungen</string>
-    <string name="pref_never_reboot">Neustartdialog verstecken</string>
-    <string name="pref_never_reboot_summary">Nach Änderungen an der Hosts-Datei wird der Dialog mit der Frage nach einem Android-Neustart nicht mehr eingeblendet.</string>
+    <string name="pref_header_preferences_summary">Allgemeine Einstellungen für AdAway</string>
+    <string name="pref_never_reboot">Neustart Erinnerung ausblenden</string>
+    <string name="pref_never_reboot_summary">Blendet den Dialog aus, der nach einem Neustart Android fragt bei erfolgreicher Anwendung</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Better: `Blendet den Neustart-Dialog nach erfolgreicher Änderung der Hosts-Datei aus`

File: AdAway/src/main/res/values-de/strings.xml [hunk 23]
```diff
@@ -195,44 +193,44 @@
...
     <string name="pref_webserver_on_boot">Webserver-Start bei Gerätestart</string>
-    <string name="pref_webserver_on_boot_summary">Beim Gerätestart den Webserver auf der lokalen IP-Adresse starten</string>
+    <string name="pref_webserver_on_boot_summary">Starten Sie den Webserver beim Hochfahren und hören Sie die lokalen IP-Adressen ab</string>
     <string name="pref_webserver_enabled">Webserver aktivieren</string>
-    <string name="pref_webserver_enabled_summary">Den lokalen Webserver aktivieren, um Anfragen an blockierte Hostnamen zu beantworten.</string>
+    <string name="pref_webserver_enabled_summary">Ermöglicht dem Webserver, lokale IP-Adressen abzufragen, um Anfragen an blockierte Hostnamen zu beantworten</string>
     <string name="pref_webserver">Lokaler Webserver</string>
     <string name="pref_custom_target">Benutzerdefiniertes Ziel</string>
-    <string name="pref_custom_target_summary">Wird nur benutzt, wenn die Hosts-Datei auf benutzerdefiniert gestellt wurde</string>
+    <string name="pref_custom_target_summary">Wird nur verwendet, wenn die Zielhostdatei auf ein Benutzerdefiniertes Ziel festgelegt ist</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: More confusing than original. Should revert.

File: AdAway/src/main/res/values-de/strings.xml [hunk 24]
```diff
@@ -241,25 +239,25 @@
...
     </string-array>
 
-    <string name="pref_update_check_summary">Beim Starten nach Aktualisierungen suchen</string>
-    <string name="pref_sdcard_problem">Diese Funktion funktioniert nicht richtig, wenn AdAway auf der SD-Karte installiert ist!</string>
+    <string name="pref_update_check_summary">Beim Starten von AdAway nach Aktualisierungen suchen</string>
+    <string name="pref_sdcard_problem">Diese Funktion funktioniert nicht richtig, wenn AdAway auf der Speicherkarte installiert ist!</string>
     <string name="pref_background">Hintergrunddienste</string>
     <string name="pref_update_check_daily">Täglich nach Aktualisierungen suchen</string>
-    <string name="pref_update_check_daily_summary">Täglich, mit Benachrichtigung, im Hintergrund nach Aktualisierungen suchen.</string>
+    <string name="pref_update_check_daily_summary">Schauen Sie täglich im Hintergrund nach Benachrichtigungen</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: ?? This is not even a german sentence. Revert.

File: AdAway/src/main/res/values-de/strings.xml [hunk 25]
```diff
@@ -241,25 +239,25 @@
...
     <string name="pref_redirection_rules">Weiterleitungen erlauben</string>
-    <string name="pref_redirection_rules_summary">SICHERHEITSGEFÄHRDEND! Weiterleitungseinträge aus Hosts-Quellen zulassen.</string>
+    <string name="pref_redirection_rules_summary">BESTEHENDES SICHERHEITSRISIKO! Zulassen von Umleitungseinträgen von Host-Quellen.</string>
     <string name="pref_debug">Fehlerdiagnose</string>
-    <string name="pref_enable_debug">Fehlerdiagnoseprotokollierung</string>
-    <string name="pref_enable_debug_summary">Sie müssen AdAway neustarten, damit diese Einstellung genutzt werden kann.</string>
-    <string name="pref_enable_ipv6">IPv6 aktivieren</string>
-    <string name="pref_enable_ipv6_summary">Alle Einträge in der Host-Datei werden einen entsprechenden Eintrag ::1 (IPv6 localhost) erhalten.</string>
+    <string name="pref_enable_debug">Fehlerdiagnose-Protokollierung</string>
+    <string name="pref_enable_debug_summary">Sie müssen AdAway Neu Starten, damit diese Einstellung wirksam wird.</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again `neu starten`

File: AdAway/src/main/res/values-de/strings.xml [hunk 26]
```diff
@@ -241,25 +239,25 @@
...
+    <string name="pref_enable_ipv6">Aktiviere IPv6</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Again the infinitive is the way to go. Revert.

File: AdAway/src/main/res/values-de/strings.xml [hunk 27]
```diff
@@ -241,25 +239,25 @@
...
+    <string name="pref_enable_ipv6_summary">Alle Host-Dateieinträge erhalten einen entsprechenden :: 1 (IPv6 localhost) Eintrag.</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: `Hosts-Dateieinträge`

File: AdAway/src/main/res/values-de/strings.xml [hunk 28]
```diff
@@ -241,25 +239,25 @@
...
     <string name="pref_enable_systemless_summary">Verhindere, dass AdAway die Systempartition ändert</string>
-    <string name="pref_enable_systemless">Systemlosen Modus aktivieren</string>
+    <string name="pref_enable_systemless">Aktiviere Systemlosen Modus</string>
```
Comment (mroettgen) [inline AdAway/src/main/res/values-de/strings.xml]: Infinitive is the way to go. Revert.

**Comment by PerfectSlayer** [comment]
Hi @squeezer11 and thanks for your translation 👍 

1. Could you accept @Vankog request and remove the files he ask? (it's an issue which is now fixed in the default locale file: 4650fab394dace508b247570efa7d12cbaae58ed)
2. I will wait @mroettgen and you agree on the last details to merge it 😉

**Comment by Vankog** [comment]
we have another contester ;-)
#1026

**Pull request closed without merging**